src/Etterna/Actor/Gameplay/Player.cpp

```diff
@@ -2325,8 +2323,6 @@ Player::FlashGhostRow(int iRow)
 {
 	const auto lastTNS =
 	  NoteDataWithScoring::LastTapNoteWithResult(m_NoteData, iRow).result.tns;
-	const auto bBlind =
```

Comment:
instead of removing this it needs to be fixed because blind is functional

Reply:
The variable was unused. That was the warning this fixed. It was used before in the line below, but that was changed in this commit: https://github.com/etternagame/etterna/commit/7d80865238aad48197dadc831b07087df420b7fd which removed the usage and left the unused local variable. Could you elaborate on what needs fixing? . I figured it was fine to just remove the local. To quote the commit comment:
> Blind should only cause your gameplay to be "blind" as in "you have no reliable feedback"

Reply:
blind removes judgment based ghost row flash feedback, turning all flashes into W1. we can go back to doing that or we can disable flashing altogether

Reply:
tested ingame this works fine as is, variable can be removed